modules/sievefilters/hm-sieve.php

```diff
@@ -2,10 +2,10 @@
 <?php
 
 class Hm_Sieve_Client_Factory {
-    public function init($user_config = null, $imap_account = null)
+    public function init($user_config = null, $imap_account = null, $sieve_tls=false)
```

Comment:
No need to add $sieve_tls parameter as it is already in $imap_account

Reply:
$imap_account has not that tls param I can add it thought as it's an array